packages/core/src/schema/utils.ts

```diff
@@ -71,3 +69,10 @@ export const schemaToEnums = (schema: Schema): { [enumName: string]: Enum } => {
 export const extractReferenceTable = (ref: ReferenceColumn): string => {
   return ref[" reference"].split(".")[0];
 };
+
+export const encodeSchema = (schema: Schema) => {
```

Comment:
Do we need a corresponding `decodeSchema`?

Reply:
At first I didn't have it because the postgres and sqlite return types are different